consensus/hotstuff/timeoutcollector/timeout_cache.go

```diff
@@ -0,0 +1,108 @@
+package timeoutcollector
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/onflow/flow-go/consensus/hotstuff/model"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	// RepeatedTimeoutErr is emitted, when we receive a timeout object for the same block
+	// from the same voter multiple times. This error does _not_ indicate
+	// equivocation.
+	RepeatedTimeoutErr              = errors.New("duplicated timeout")
+	TimeoutForIncompatibleViewError = errors.New("timeout for incompatible view")
```

Comment:
```suggestion
	// ErrRepeatedTimeout is emitted, when we receive a timeout object for the same block
	// from the same voter multiple times. This error does _not_ indicate
	// equivocation.
	ErrRepeatedTimeout              = errors.New("duplicated timeout")
	ErrTimeoutForIncompatibleView   = errors.New("timeout for incompatible view")
```
Nitpick: sentinel error naming convention. Makes it easier to distinguish sentinel values and error types.